lib/Capabilities.php

```diff
@@ -182,6 +192,15 @@ public function getCapabilities(): array {
 		}
 
 		if ($user instanceof IUser) {
+			if ($this->talkConfig->isFederationEnabled() && $this->talkConfig->isFederationEnabledForUserId($user)) {
+				$capabilities['config']['federation'] = [
+					'enabled' => true,
+					'incoming' => $this->appConfig->getAppValueBool('federation_incoming_enabled', true),
+					'outgoing' => $this->appConfig->getAppValueBool('federation_outgoing_enabled', true),
+					'trusted-servers' => $this->appConfig->getAppValueBool('federation_only_trusted_servers'),
```

Comment:
```suggestion
					'incoming-enabled' => $this->appConfig->getAppValueBool('federation_incoming_enabled', true),
					'outgoing-enabled' => $this->appConfig->getAppValueBool('federation_outgoing_enabled', true),
					'only-trusted-servers' => $this->appConfig->getAppValueBool('federation_only_trusted_servers'),
```

Just my opinion to be a bit more verbose here?